### PR TITLE
chore: Reuses project in tests for `cluster` resource

### PR DESCRIPTION
**Pull request by lantoli**
## Description

Reuses project in tests for `cluster` resource

Link to any related issue(s): CLOUDP-237988

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [X] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [X] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [X] I have checked that this change does not generate any credentials and that **they are NOT accidentally logged anywhere**.
- [X] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [X] I have added any necessary documentation (if appropriate)
- [X] I have run make fmt and formatted my code
- [X] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.
- [X] If changes include removal or addition of 3rd party GitHub actions, I updated our internal document. Reach out to the APIx Integration slack channel to get access to the internal document.

### CODE REVIEW

File: internal/service/cluster/resource_cluster_migration_test.go [hunk 1]
```diff
@@ -85,40 +76,36 @@ func TestMigClusterRSCluster_basic_PartialAdvancedConf_backwardCompatibility(t *
...
+			project_id   = %[1]q
+			name         = %[2]q
 			disk_size_gb = 10
 
             cluster_type = "REPLICASET"
 		    replication_specs {
 			  num_shards = 1
 			  regions_config {
-			     region_name     = "EU_CENTRAL_1"
+			     region_name     = "US_WEST_2"
```
Comment (oarbusi) [inline internal/service/cluster/resource_cluster_migration_test.go:90]: I am curious about the change of region, why the change?

**Review by oarbusi — APPROVED** [review]
LGTM

File: internal/service/cluster/resource_cluster_migration_test.go (hunk 1 above)
Comment (lantoli) [reply internal/service/cluster/resource_cluster_migration_test.go:90]: to avoid cross-region limits
